docs/plus/index.md

```diff
@@ -0,0 +1,338 @@
+# cdk8s+ v25
```

Comment:
Why did you add this file here? It exists in the website repo

Reply:
Just removed it. Forgot we moved this back to cdk8s.